source/deployment/sso-ldap.md

```diff
@@ -53,7 +53,7 @@ To configure AD/LDAP synchronization with AD/LDAP sign-in:
 
 1. Go to **System Console > Authentication > AD/LDAP** and set **Enable Synchronization with AD/LDAP** to `true`.
 
-2. Scroll down to **Syncronization Interval (minutes)** to specify how often Mattermost accounts synchronize attributes with AD/LDAP. The default setting is 60 minutes.
+2. Scroll down to **Syncronization Interval (minutes)** to specify how often Mattermost accounts synchronize attributes with AD/LDAP. The default setting is 60 minutes. *note profile picture attribute is synchronized upon user login only)*
```

Comment:
Missing a parenthesis or have an extra.  Should `Note:` be capitalized with comma or colon?